tools/builder.js

```diff
@@ -19,7 +19,7 @@ colors.setTheme({
  * @param {RegExp[]} opts.include
  */
 function string(opts) {
-  const minifier = new CleanCSS();
+  const minifier = new CleanCSS({ format: "keep-breaks" });
```

Comment:
This will increase the bundle size slightly (more newlines), but might be worth it.